cmake/onnxruntime_providers.cmake

```diff
@@ -1030,8 +1030,7 @@ if (onnxruntime_USE_ROCM)
   # Generate GPU code for GFX9 Generation
   list(APPEND HIP_CLANG_FLAGS --amdgpu-target=gfx906 --amdgpu-target=gfx908)
 
-  hip_add_library(onnxruntime_providers_rocm ${onnxruntime_providers_rocm_src})
-
+  hip_add_library(onnxruntime_providers_rocm ${onnxruntime_providers_rocm_src} HIPCC_OPTIONS -std=c++14 NVCC_OPTIONS -std=c++14)
```

Comment:
Could you add the line below to line#1019 rather than change here? 

list(APPEND HIP_CXX_FLAGS -std=c++14)

Reply:
Updated.

Reply:
Thanks!